sql/core/src/test/scala/org/apache/spark/sql/ParsingImprovementsSuite.scala

```diff
@@ -0,0 +1,183 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql
+
+import java.util.UUID
+
+import org.apache.spark.sql.{DataFrame, QueryTest}
+import org.apache.spark.sql.internal.SQLConf
+import org.apache.spark.sql.test.SharedSparkSession
+
+class ParsingImprovementsSuite extends QueryTest with SharedSparkSession {
```

Comment:
```suggestion
class InlineTableParsingImprovementsSuite extends QueryTest with SharedSparkSession {
```

Reply:
Looks like I'm not able to import many of the dependencies we have in this file from `ResolveInlineTableSuites`

Specifically getting:

```
[ERROR] [Error] /home/costas.zarifis/spark/sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/ResolveInlineTablesSuite.scala:31: object DataFrame is not a member of package org.apache.spark.sql
[ERROR] [Error] /home/costas.zarifis/spark/sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/ResolveInlineTablesSuite.scala:33: object test is not a member of package org.apache.spark.sql
did you mean types?

```

For the following imports:
```
import org.apache.spark.sql.{DataFrame, QueryTest}
import org.apache.spark.sql.test.SharedSparkSession
```